modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/EditNotificationTemplate.tsx

```diff
@@ -397,14 +409,16 @@ interface IProps {
 	notificationTemplateId: number;
 }
 
-type TNotificationTemplate = {
+export type TNotificationTemplate = {
+	attachmentObjectFieldIds: string[] | number[];
```

Comment:
Can you explain me why it can be both types?

Reply:
Hi @rebsilva, the attachmentObjectFieldIds comes from back end as an array of Number: `attachmentObjectFieldIds: [41618]`
But the options of `<FormCustomSelect>`, which extends from `CustomItem`, only accepts values as String. I was not able to make this work with Number only. If you find a better way to do this feel free to share. :smiling_face_with_three_hearts:

Reply:
I see! One way of doing it is to transform all the `attachmentObjectFieldIds` values into Strings, but it would cause too much trouble. I was worried about the principles of the Typescript to not set a lot of types to the same variable since it kinda loses the meaning of using Typescript in the first place, but in this case I think this is fine! Thanks a lot for explaining to me! :heart:

Reply:
Yes. I completely understand your point and this types needs to be fixed. Let's just wait a safer moment to make this change. hehehe